demo/kubernetes/README.md

```diff
@@ -4,74 +4,117 @@ A demo of persistent block storage from rook within a [Kubernetes](http://kubern
 
 ## Prerequisites
 
-A running Kubernetes cluster is required. See the <https://github.com/rook/coreos-kubernetes/tree/rook-demo> (rook-demo branch) in the rook fork of the coreos-kubernetes repo.
-The changes to coreos-kubernetes necessary to support the scenario include:
-1. Devices created in the vagrant environment to provide storage in the cluster.
-2. Access to modprobe, necessary for using the rbd volume plugin (<https://github.com/kubernetes/kubernetes/issues/23924>).
+A running Kubernetes cluster is required. First clone <https://github.com/rook/coreos-kubernetes/tree/rook-demo> (rook-demo branch in the rook fork of the coreos-kubernetes repo):
+```
+git clone https://github.com/rook/coreos-kubernetes.git
+cd coreos-kubernetes
+git checkout rook-demo
+```
+
+The changes found in that branch that facilitate this demo scenario include:  
+1. Devices created in the vagrant environment to provide storage in the cluster.  
+2. Access to modprobe, necessary for using the rbd volume plugin (<https://github.com/kubernetes/kubernetes/issues/23924>).  
 
 ### Multi-node deployment
 
 If you are using the rook-demo branch you can get a vagrant kubernetes cluster running like this (from the root of the repo):
-
-    cd multi-node/vagrant
-    vagrant up
-    export KUBECONFIG="$(pwd)/kubeconfig"
-    kubectl config use-context vagrant-multi
-
-Verify that kubernetes is done initializing.
-
-    kubectl get nodes
-
+```
+cd multi-node/vagrant
+vagrant up
+export KUBECONFIG="$(pwd)/kubeconfig"
+kubectl config use-context vagrant-multi
+```
+
+Then wait for the cluster to come up and verify that kubernetes is done initializing (be patient, it takes a bit):
+```
+kubectl get nodes
+```
 
 ### Single-node deployment
 
-This walkthrough is using the multi-node deployment. To use the single-node deployment, the main change to the instructions is the ip address.
+This walkthrough is using the multi-node deployment. To instead use the single-node deployment, the main change to the instructions is the IP address.
 Replace `172.17.4.101` with `172.17.4.99` in these instructions and in `rook.yml`.
 
 ## Starting rook
-
-    cd <rook>/demo/kubernetes
-    export TOKEN=$(curl -s -w "\n" 'https://discovery.etcd.io/new?size=1')
-    kubectl create configmap rookd --from-literal=discovery-token=$TOKEN
-    kubectl create -f rook.yml
+```
+cd <rook>/demo/kubernetes
+export TOKEN=$(curl -s -w "\n" 'https://discovery.etcd.io/new?size=1')
+kubectl create configmap rookd --from-literal=discovery-token=$TOKEN
+kubectl create -f rook.yml
+```
```

Comment:
i prefer the triple backtick syntax best because it's not whitespace sensitive.  the new blocks i added were using that pattern, so then i just changed all the others for consistency.